README.adoc

```diff
@@ -132,20 +115,8 @@ is to install following packages:
 - ovirt-engine-wildfly
 - ovirt-engine-wildfly-overlay
 
-Both packages can be installed from ovirt-master-snapshot-static repository:
-
-  [ovirt-master-snapshot-static]
-  name=Latest oVirt master additional nightly snapshot
-  baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/@DIST@$releasever/
-  enabled=1
-  skip_if_unavailable=1
-  gpgcheck=0
-
-Please replace `@DIST@` with `fc` for Fedora or `el` for Centos/RHEL.
-
-Alternatively, repository list can be updated.
-
-- For CentOS Stream 8 use:
+Both packages can be installed from CentOS Stream 8 repositories.
```

Comment:
I'd say "from oVirt COPR", these are not standard Centos Stream 8 repos. And I wouldn't say Stream 8 either, it should provide all the deps on el9 too once we complete el9 support...